### PR TITLE
Initial jsonld context definition plus example; closes #17

### LINKED ISSUE #17: JSON-LD Context for PCDM (and related)
From https://jira.duraspace.org/browse/FCREPO-1415 we should provide a JSON-LD context document for ease and consistency of development and serialization.

### PR DESCRIPTION
**Pull request by azaroth42**
Closes #17, test case in the jsonld playground:
    http://tinyurl.com/pjnbtca

### CODE REVIEW

**Comment by awoods** [comment]
Looks good to me.

**Comment by awoods** [comment]
Other committers, thoughts on this PR?
Also, this issue is being tracked here: https://jira.duraspace.org/browse/FCREPO-1415

**Comment by escowles** [comment]
With the caveat that I don't know much about JSON-LD, this looks good to me.

**Comment by acoburn** [comment]
Shouldn't the context file should also contain the new inverse properties: memberOf and the like?

**Comment by azaroth42** [comment]
@acoburn Added! Thanks :)

File: context.json [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+		"hasMember": {"@id": "pcdm:hasMember", "@type":"@id"},
+		"hasRelatedObject": {"@id": "pcdm:hasRelatedObject","@type":"@id"},
+		"hasFile": {"@id": "pcdm:hasFile","@type":"@id"},
+		"hasRelatedFile": {"@id": "pcdm:hasRelatedFile","@type":"@id"},			
+
+		"memberOf": {"@id": "pcdm:memberOf", "@type:""@id"},
+		"fileOf": {"@id": "pcdm:fileOf", "@type:""@id"},	
+		"relatedObjectOf": {"@id": "pcdm:relatedObjectOf", "@type:""@id"},			
+		"relatedFileOf": {"@id": "pcdm:relatedFileOf", "@type:""@id"},
+
```
Comment (acoburn) [inline context.json:27]: The JSON formatting is incorrect: invert the :"

File: examples/example.jsonld [hunk 2]
```diff
@@ -0,0 +1,25 @@
+
+{
+  "@context": "http://pcdm.org/context-pcdm.json",
```
Comment (acoburn) [inline examples/example.jsonld]: You'll want to make sure the context value aligns with the name of the context file.

**Comment by azaroth42** [comment]
FIxed and squashed minor commits.

**Pull request merged**